core/src/css/ionic-swiper-element.scss

```diff
@@ -0,0 +1,38 @@
+@import "../components/slides/slides.ios.vars.scss";
```

Comment:
Threading for discussion:

How would you feel about not shipping this in Ionic but instead keeping it as an optional code snippet on the migration guide?

A couple reasons:

1. One less "public API" we need to maintain, and we can remove the other Swiper stylesheet in a future major release.
2. If Swiper 10 comes out and makes a breaking change to styling, we'll need to scramble to update this stylesheet and do it in a way such that both Swiper 9 and 10 are supported. At that point, we'll be back to having to maintain some level of Swiper support.

-----

Another thought: We could also provide a migration table for the CSS variables like we do for other Swiper APIs (example: https://ionicframework.com/docs/angular/slides#events)

This table would show the Ionic CSS variable and the equivalent Swiper CSS variable. That way developers who are customizing Swiper can have a similar style result without having to use Ionic-created variables.

Reply:
Makes sense to me to have the CSS available in the docs instead of us managing and shipping another stylesheet 👍